nautiluszim/scraper.py

```diff
@@ -342,29 +344,75 @@ def test_collection(self):
             all_names = zh.namelist()
 
         missing_files = []
+        saved_file_paths = []
```

Comment:
```suggestion
        all_file_paths = []
```

`saved` mislead reading into thinking it's a special list